src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -175,8 +181,14 @@ public String convert(Object o) {
 	public Class<?>[] acceptChange(ChangeMode mode) {
 		if (mode == ChangeMode.SET || mode == ChangeMode.RESET) {
 			if (mark == 1 && Player.class.isAssignableFrom(getExpr().getReturnType())) {
-				Skript.error("Can't change the Minecraft name of a player. Change the 'display name' or 'tab list name' instead.");
-				return null;
+				if(!PLAYER_NAME_CHANGEABLE) {
```

Comment:
Spaces after `if` ;D

Reply:
I really wonder why I'm not allowed to make whitespace changes not related to the pull request. Right after that condition there is 
```java
				if(mode == ChangeMode.RESET) {
					Skript.error("Can't reset the Minecraft name of a player.");
					return null;
				}
``` which doesn't have a space after if

Reply:
What do you mean?

Reply:
I've been told 'line changes should only be done for things relevant to the theme of the PR.' many times

Reply:
I mean, you added that part yourself. You are good to change it.